lib/daru/accessors/array_wrapper.rb

```diff
@@ -67,26 +67,24 @@ def dup
       end
 
       def mean
-        sum.quo(@size - @context.missing_positions.size).to_f
+        return nil if @data.compact.empty?
```

Comment:
You can add a `#compact` inside the wrappers though. Make sure you have an implementation for each wrapper.